web/pageComponents/shared/portableText/RichText.tsx

```diff
@@ -44,8 +44,16 @@ const defaultSerializers = {
   },
   marks: { sub: Sub, sup: Sup, link: ExternalLink, internalLink: InternalLink },
   list: {
-    bullet: BulletList,
-    number: NumberedList,
+    bullet: ({ children }: PortableTextBlock) => (
```

Comment:
new code is the same as before, but more verbose. better to have it as it was before